tests/e2e/support/commands.js

```diff
@@ -97,3 +97,43 @@ Cypress.Commands.add('addLineItem', (url, quantity) => {
   cy.get('[data-test=add-to-cart-form-button]').click();
   cy.get('[data-test=mini-cart-content]').should('be.visible');
 });
+
+Cypress.Commands.add('createMyOrder', (draft) => {
+  const createNewOrder = client => client.query({
+    query: gql`
+      query queryCustomerByEmail($predicate: String) {
+        customers(limit: 1, where: $predicate) {
+          results {
+            id
+            version
+          }
+        }
+      }`,
+    variables: { predicate: `email = "${draft.customerEmail}"` },
+    fetchPolicy: 'network-only',
+  }).then(async (response) => {
+    const customer = response.data.customers.results[0];
+    return client.mutate({
+      mutation: gql`
+      mutation createMyCart($draft: CartDraft!){
+      createCart (draft: $draft) {
+        id, version
+      }
+  }`,
+      variables: { draft: { ...draft, customerId: customer.id } },
+    });
+  }).then((response) => {
+    const cart = response.data.createCart;
+    const orderNumber = function orderNumber() { return Math.floor(Math.random() * 100000).toString(); };
```

Comment:
This piece of code is going to be executed again on every order creation, so the order number is going to be different every time. Or it should be. Did you find a situation in which the order number was the same and that's why you had to make it a function?

Reply:
When creating order it doesn't create order number, it has to be added manually,
that's why I decided to have a random number there

Reply:
The random number part is fine. What I mean is that you don't need to create a function for it.